build_package.py

```diff
@@ -1234,6 +1233,31 @@ def create_scriptlets():
     fp.write(
         r"""#!/bin/bash
 
```

Comment:
Let's add a docstring on why we need to do that (for compatibility with RHEL 8.1 and other distros where package name is python2 until we have per distro packages).